### PR TITLE
Add snapshotter CRDs after cluster setup

### PR DESCRIPTION
**Pull request by ggriffiths**
```release-notes
Adds snapshotter CRDs after cluster setup
```

Changes required for https://github.com/kubernetes-csi/csi-driver-host-path/pull/98

Signed-off-by: Grant Griffiths <grant@portworx.com>

### CODE REVIEW

File: prow.sh [hunk 1]
```diff
@@ -987,6 +1047,16 @@ main () {
         if tests_need_non_alpha_cluster; then
             start_cluster || die "starting the non-alpha cluster failed"
 
+            # Install necessary snapshot CRDs and snapshot controller
+            # For Kubernetes 1.17+, we will install the CRDs and snapshot controller.
+            if version_gt "${CSI_PROW_KUBERNETES_VERSION}" "1.16.255" || "${CSI_PROW_KUBERNETES_VERSION}" == "latest"; then
```
Comment (xing-yang) [inline prow.sh]: Is "1.16.255" 1.17?

Comment (ggriffiths) [reply prow.sh]: Yes, I couldn't find a better way to do version comparisons in bash. Open to suggestions on that

File: prow.sh [hunk 2]
```diff
@@ -657,6 +660,59 @@ install_hostpath () {
...
+  CRD_BASE_DIR="https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}/config/crd"
+  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshotclasses.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshots.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshotcontents.yaml" --validate=false
+  cnt=0
+  until kubectl get volumesnapshotclasses.snapshot.storage.k8s.io \
+    && kubectl get volumesnapshots.snapshot.storage.k8s.io \
+    && kubectl get volumesnapshotcontents.snapshot.storage.k8s.io; do
+    if [ $cnt -gt 30 ]; then
+        echo >&2 "ERROR: snapshot CRDs not ready after over 1min"
```
[the remark this answers is not in the document]
Comment (ggriffiths) [reply prow.sh]: Fixed these :+1:

**Comment by xing-yang** [comment]
@pohly can you approve it?  I don't have permissions.

**Comment by pohly** [comment]
Next steps:
- merge into csi-driver-host-path repo via `git submodule pull`
- do a release of csi-driver-host-path
- update `CSI_PROW_HOSTPATH_VERSION` to that version in `csi-release-tools`
- import into all repos
- enable "1.17 on 1.17" pre-merge jobs

**Pull request merged**